scalar_test.go

```diff
@@ -221,3 +224,146 @@ func TestNewIdentifierArray(t *testing.T) {
 	autopilot.Equals(t, "my-service", *result[0].Alias)
 	autopilot.Equals(t, ol.ID("Z2lkOi8vMTIzNDU2Nzg5"), *result[1].Id)
 }
+
+func TestNullValueString(t *testing.T) {
```

Comment:
Optionally since we know we only want `Nullable` for string - you could add a type constraint if you are worried about people trying to use this for other types.  That way its future flexible to expand out to other types when we are ready.

Reply:
@rocktavious pushed a commit to add the type constraint so that only `string` is supported for now. Like you said we can expand the constraint and add the extra tests later.